### DISCUSSION TITLE
Release Testing 2023.05.1 - RC3

### DISCUSSION THREAD

**Discussion by tedpatrick** [pyscript/pyscript#1541] [Announcements]
We are feature complete for a release of PyScript. We need your help testing 2023.05.1.

How to test 2023.05.1.RC3

```
<link rel="stylesheet" href="https://pyscript.net/snapshots/2023.05.1.RC3/pyscript.css"/>
<script defer src="https://pyscript.net/snapshots/2023.05.1.RC3/pyscript.js"></script>
```

Thank you for your ongoing support and contribution to PyScript.

2023.05.1.RC3 Example:

```
<html>
<head>
    <link rel="stylesheet" href="https://pyscript.net/snapshots/2023.05.1.RC3/pyscript.css" />
    <script defer src="https://pyscript.net/snapshots/2023.05.1.RC3/pyscript.js"></script>
</head>
<body>
    <py-config type="toml">
packages = [
    "pandas"
]
    </py-config>
    <py-script>
import js
import pandas as pd

js.console.log( "2023.05.1.RC3" )
js.document.title = "2023.05.1.RC3"
print( "2023.05.1.RC3" )
print( pd.show_versions() )
    </py-script>
</body>
</html>
```

**Comment by bugzpodder** [comment]
pyodide 0.23.3 is released: https://pyodide.org/en/stable/project/changelog.html#version-0-23-3

**Comment by JeffersGlass** [comment]
Doing `pyscript.__version__` still shows `2022.12.1.dev` 

https://github.com/pyscript/pyscript/blob/848d77b1c29d4145a3438b832acd4b3b9bf32951/pyscriptjs/src/version.ts#L1-L10

**Comment by Neon22** [comment]
working well for me but is @when missing ?

this:
```
@when("onpointerup", selector=".scrubbable-num")
def nodrag(evt):
    evt.target.dragging = None
```
gives me this error
```
File "<exec>", line 253, in <module>
NameError: name 'when' is not defined
```
I am running:
Running Pyiodide version: 0.23.2 Pyscript verions: 2022.12.1.dev

**Reply by JeffersGlass** [reply]
You’ll need to add `from pyscript import when`

**Reply by JeffersGlass** [reply]
Also, the name of the event type is just “pointerup”, not “onpointerup”

**Reply by Neon22** [reply]
sigh... LOL - I will raise a pr for the docs page. Thanks

**Comment by mahmoud** [comment]
Given that 2023.11.01 is out now, might want to unpin/close this discussion.

**Comment by JeffersGlass** [comment]
Agreed! Thanks @mahmoud .

**Comment by WebReflection** [comment]
Unpinned and closed.
